test/uk/gov/hmrc/economiccrimelevyregistration/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -443,4 +448,35 @@ class CheckYourAnswersControllerSpec extends SpecBase {
         }
     }
   }
+
+  "createAndEncodeHtmlForPdf" should {
+    "show liability start date" in forAll(
```

Comment:
can we have a test for when the date is not present? How do we want to handle that?